src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/entities/ListingEvent.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.iahearingsapi.domain.entities;
+
+import static java.util.Arrays.stream;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Optional;
+
+public enum ListingEvent {
```

Comment:
I have added this class already in the initialListing. sub bail package

Reply:
Ah I just noticed. I added it before rebasing. Now I've removed this.